server/rooms.ts

```diff
@@ -328,7 +328,7 @@ export abstract class BasicRoom {
 		if (this.userCount) Sockets.roomBroadcast(this.roomid, message);
 	}
 	sendMods(data: string) {
-		this.sendRankedUsers(data, '%');
+		this.sendRankedUsers(data, '*');
```

Comment:
This will also send private mod actions to Section Leaders. Is that okay?

Reply:
They should already be getting them in theory, since anyone who has access to the staff room gets notified by this currently (which is a different bug, but still fine imo).

Reply:
>  (which is a different bug, but still fine imo)

fwiw, not a bug - staff room access grants `user.isStaff`, which intentionally gives access to all private mod actions.